src/HttpClientFactory/Polly/src/Microsoft.Extensions.Http.Polly.csproj

```diff
@@ -13,7 +13,6 @@
     <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
     <IsPackable>true</IsPackable>
     <IsShipping>true</IsShipping>
-    <Nullable>disable</Nullable>
```

Comment:
You didn't actually annotate the polly library 😕 
For example, `HttpRequestMessageExtensions.GetPolicyExecutionContext` can return a null `Context` and `HttpRequestMessageExtensions.SetPolicyExecutionContext` can accept a null `Context`.

If we aren't going to annotate it yet, then we shouldn't remove this.

Reply:
Hmm, I didn't get any warnings after enabling nullable. I guess you can enable nullable for netstandard2.0, but there is no analysis.

Reply:
Done